store-implementation/proxy-store/src/main/java/uk/gov/gchq/gaffer/proxystore/ProxyStore.java

```diff
@@ -80,6 +80,10 @@
     private Schema schema;
     private Set<Class<? extends Operation>> supportedOperations;
 
+    public ProxyStore() {
```

Comment:
This will be a breaking change. For anyone using it currently it will support the core op handlers, if we add this it will now not.